Changed default node labels (#33)
stfc/cloud-deployed-apps: docs/DEPLOYING_APPS.md

# Pre-deployment steps

> # 1. Label nodes to run longhorn on

Reviewer: ```suggestion
# 1. (Optional) Label nodes to run longhorn on
```

> Make sure you have labelled your nodes so that longhorn can use them as storage nodes. You want to label your worker nodes, the default label is `longhorn.demo.io/longhorn-storage-node=true` but you can change this in the cluster-specific values like so:	

Reviewer: ```suggestion
Make sure you have labelled your nodes so that longhorn can use them as storage nodes. 

If you're also managing `capi`, these are set for you - so you don't need to do anything.  

You want to label your worker nodes, the default label is `longhorn.demo.io/longhorn-storage-node=true`  

You can change this in the cluster-specific values like so:	
```

```	
longhorn:	
  longhornManager:	
    nodeSelector: 	
      # change this to whatever label you want	
      longhorn.store.nodeselect/longhorn-storage-node: true	
```	

> **NOTE:** If you're using `capi` or any other `infra` that you're also managing with argocd - make sure you set these labels accordingly for your cluster. 	

Reviewer: ```suggestion
**NOTE:** If you're NOT using `capi` - make sure you set these labels accordingly for your cluster. 	
```

> For `capi` you can set node labels like so:	

Reviewer: ```suggestion
If you want to change the node labels that capi uses - you can do so like this: 
```

# Pre-deployment steps

> ## 1. Label nodes to run longhorn on 

Reviewer: I'd still keep this - just make it clear that you'd only need this if you use something other than CAPI to deploy your cluster
